include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -266,8 +266,19 @@ typedef struct OrtCUDAProviderOptions {
   size_t cuda_mem_limit;                          // default cuda memory limitation to maximum finite value of size_t.
   int arena_extend_strategy;                      // default area extend strategy to KNextPowerOfTwo.
   int do_copy_in_default_stream;
+  int has_user_compute_stream;
```

Comment:
Is this a dup of user_compute_stream when nullptr means no user compute stream?

Reply:
has_user_compute_stream = false means no user compute stream.  has_user_compute_stream = true means to use user specified stream, which may be default stream if it is nullptr.

Reply:
OK, so default in ORT != default in CUDA

Reply:
Exactly. Thanks!